dir.props

```diff
@@ -10,7 +10,7 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00134</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00137</BuildToolsVersion>
```

Comment:
Should we go up to 141 to match CoreFx?  As long as 137 doesn't break TFS builds, we can do that at a later time.